### PR TITLE
Feature images

### PR DESCRIPTION
**Pull request by malczak**
Adds 
* page avatar
* post publish date formatting
* post images

image

### CODE REVIEW

File: builders/buildPage.ts [hunk 1]
```diff
@@ -3,22 +3,35 @@ import Page from "../models/Page";
...
-}
+};
+
+const buildAvatar = ($: CheerioSelector) => {
+  const ogi = $('meta[property="og:image"]');
+  let avatar = ogi && ogi.attr("content");
+
+  if (!avatar) {
+    const ai = $('a[rel="theater"] img');
+    avatar = ai && ai.first().attr("src");
```
Comment (sobstel) [inline builders/buildPage.ts]: `avatar = ai.attr("src")` should be fine. It works like jQuery, should call `attr` on a first element only and return a value

File: builders/buildPage.ts [hunk 2]
```diff
@@ -3,22 +3,35 @@ import Page from "../models/Page";
...
+  const lastDashIndex = title.lastIndexOf("-");
   if (lastDashIndex !== -1) {
     name = title.substring(0, lastDashIndex - 1);
   }
 
   return name;
-}
+};
+
+const buildAvatar = ($: CheerioSelector) => {
```
Comment (sobstel) [inline builders/buildPage.ts:27]: For now it's fine, but name and avatar should be actually read from post itself. 
Is it actually possible it's posted by someone else on the fb page???

File: models/Post.ts [hunk 3]
```diff
@@ -17,6 +20,14 @@ class Post {
   get timestamp(): number {
     return this.input.timestamp;
   }
+
+  get images(): string[] {
+    return this.input.images;
+  }
+
+  getFormattedDate() {
```
Comment (sobstel) [inline models/Post.ts]: Not needed anymore. Please remove.

**Comment by sobstel** [comment]
It looks really good. Just a few minor notes and we can merge it 🚀

File: builders/buildPage.ts (hunk 1 above)
Comment (sobstel) [reply builders/buildPage.ts]: or even `avatar = $('a[rel="theater"] img').attr("src")`?

File: builders/buildPage.ts (hunk 2 above)
Comment (malczak) [reply builders/buildPage.ts:27]: I don't think so. Only page admins/editors can add posts on page behalf

File: builders/buildPage.ts (hunk 1 above)
Comment (malczak) [reply builders/buildPage.ts]: I will not throw?

Comment (sobstel) [reply builders/buildPage.ts]: Should not. I'll check it.

**Pull request merged**